contracts/L2ApplicationStateRootProvider.sol

```diff
@@ -0,0 +1,10 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
```

Comment:
Question to reviewers - what license do we want here? Is `MIT` fine?

Reply:
Monomer is Apache 2.0. Let's use that.